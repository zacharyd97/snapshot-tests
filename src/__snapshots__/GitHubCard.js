import  Button from 'react-bootstrap/Button'
import Card from 'react-bootstrap/Card'

function GitHubCard() {
    return (
        <Card style={{ width: '18rem' }}>
      <Card.Img variant="top" src="https://picsum.photos/100/180" />
      <Card.Body>
        <Card.Title>ZacharyD97</Card.Title>
        <Card.Text>
          My name is zach i am a full stack software engineer with skill in reactjs, javaScript and bootstrap
        </Card.Text>
      </Card.Body>
    </Card>
    )
}

export default GitHubCard