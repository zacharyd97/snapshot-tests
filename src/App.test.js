import { render, screen } from '@testing-library/react';
import App from './App';
import renderer from 'react-test-renderer'
import GitHubCard from './__snapshots__/GitHubCard';

test('renders a snapshot', () => {
  const tree = renderer.create(<App/>).toJSON()
  expect(tree).toMatchSnapshot()
})

test('GitHubCard stapshot', () => {
  const tree = renderer.create(<GitHubCard/>).toJSON()
  expect(tree).toMatchSnapshot()
}) 
