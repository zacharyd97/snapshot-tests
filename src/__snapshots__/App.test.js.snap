// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GitHubCard stapshot 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    className="card-img-top"
    src="https://picsum.photos/100/180"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      ZacharyD97
    </div>
    <p
      className="card-text"
    >
      My name is zach i am a full stack software engineer with skill in reactjs, javaScript and bootstrap
    </p>
  </div>
</div>
`;

exports[`renders a snapshot 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <p>
      Edit 
      <code>
        src/App.js
      </code>
       and save to reload.
    </p>
    <h2>
      Hello world
    </h2>
    <a
      className="App-link"
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      Learn React
    </a>
  </header>
</div>
`;
